thor/src/protocols/splice.rs

```diff
@@ -12,7 +12,9 @@ use crate::{
 use anyhow::{Context, Result};
 use async_trait::async_trait;
 use bitcoin::{
-    consensus::serialize, util::psbt::PartiallySignedTransaction, Address, Amount, Transaction,
+    consensus::serialize,
+    util::{amount::serde::as_sat, psbt::PartiallySignedTransaction},
```

Comment:
same, `as_sat` import should be done behind the `serde` feature to avoid build warnings when compiling without it.